src/styles/components/buttonStyles.ts

```diff
@@ -18,6 +18,17 @@ const buttonStyles = {
       _hover: { bg: "brand.secondary", opacity: "0.60" },
       _active: { bg: "brand.secondary", opacity: "0.60" },
     },
+    checkoutBtn: {
+      bg: "brand.secondary",
+      color: "white",
+      size: "lg",
+      fontSize: "lg",
+      fontWeights: "bold",
+      lineHeights: "22",
+      borderRadius: "6px",
+      _hover: { bg: "brand.secondary", opacity: "0.60" },
+      _active: { bg: "brand.secondary", opacity: "0.60" },
+    },
```

Comment:
This button seems same as "shareBtn"? Maybe making these same styles into one sharing style can be a better idea.

Reply:
> This button seems same as "shareBtn"? Maybe making these same styles into one sharing style can be a better idea.

The CheckoutBtn was merged to ShareBtn and deleted.